Classification/cnns/train.sh

```diff
@@ -2,13 +2,14 @@
 rm -rf core.* 
 rm -rf ./output/snapshots/*
 
+
 # training with synthetic data
 python3 of_cnn_train_val.py \
     --num_examples=50 \
     --num_val_examples=50 \
     --num_nodes=1 \
     --gpu_num_per_node=1 \
-    --model_update="momentum" \
+    --optimizer="adam" \
```

Comment:
suggest to keep `momentum` as the default optimizer

Reply:
momentum is no longer exists alone in new optimizer api, instead, 
a new way to use momentum is :  --optimizer="sdg"  --momentum=0.875

Reply:
This `momentum` is the option of CNN's optimizer but not oneflow api, the default resnet50 optimizer is momentum(sgd + mom) but not `adam` .

Reply:
ok,already modified